gapic/src/main/com/google/gapid/perfetto/views/TrackContainer.java

```diff
@@ -326,7 +326,9 @@ public Dragger onDragStart(double x, double y, int mods) {
 
     @Override
     public Hover onMouseMove(Fonts.TextMeasurer m, double x, double y) {
-      if (y < TITLE_HEIGHT && (expanded || x < LABEL_WIDTH)) {
+      if (y < TITLE_HEIGHT && expanded && x >= LABEL_WIDTH) {
+        return Hover.NONE;
+      } else if (y < TITLE_HEIGHT && (expanded || x < LABEL_WIDTH)) {
```

Comment:
Oops, no that doesn't quite work. If `expanded` is `true`, the title may be longer than `LABEL_WIDTH`.

Reply:
Remove this part in the new commit, will handle the deselection at empty title area after #3523 is checked in.